app/deployer/terraform/terraform.tf

```diff
@@ -0,0 +1,142 @@
+# Configure Azure provider
+provider "azurerm" {
+  subscription_id = "${var.azure_subscription_id}"
+  client_id       = "${var.azure_client_id}"
+  client_secret   = "${var.azure_client_secret}"
+  tenant_id       = "${var.azure_tenant_id}"
+}
+
+# create a resource group if it doesn't exist
+resource "azurerm_resource_group" "rg" {
+    name = "cuackcuackrg"
```

Comment:
It shouldn't be part of the repo. It gets generated each time a deployment happens. I'll remove it and add it to .gitignore.